### ISSUE TITLE
Energy Trashcan NPE

### ISSUE THREAD

**Issue by hlanuel** [Luligabi1/CoxinhaUtilities#26]
Hello, 
We've got an issue on our server using the modpack All of Fabrics 6. 
I accidentally used an Energy Trashcan instead of a basic trashcan. I used pipes to send items to the energy trashcan.
After the pipes received items the server was constantly crashing when loading the chunk where the trashcan is placed. We had no other choice than reload a backup.

Here is the error from the logs : 
```
Caused by: java.lang.NullPointerException: Cannot invoke "team.reborn.energy.api.EnergyStorage.supportsExtraction()" because "stackEnergy" is null
at me.luligabi.coxinhautilities.common.block.trashcan.energy.EnergyTrashCanBlockEntity.tick(EnergyTrashCanBlockEntity.java:53) ~[coxinhautilities-1.4.5+1.19.jar:?]
```

**Comment by hlanuel** [comment]
```
        if(stackEnergy != null & stackEnergy.supportsExtraction()) {
            EnergyStorageUtil.move(stackEnergy, blockEntity.energyStorage, Long.MAX_VALUE, null);
        }
```

We should have a && operator here

**Comment by Luligabi1** [comment]
I'm releasing a fix on 1.19, 1.19.4 and 1.20 for this asap. Thank you for your report and sorry about your world, I'll be contacting the AOF crew to let them know about this
